xarray/tests/test_units.py

```diff
@@ -19,6 +21,13 @@
 unit_registry = pint.UnitRegistry(force_ndarray=True)
 Quantity = unit_registry.Quantity
 
+
+# silence pint's BehaviorChangeWarning
+with warnings.catch_warnings():
+    warnings.simplefilter("ignore")
+    Quantity([])
```

Comment:
this is not necessary now that `pint=0.11` has been released which does not warn